libc/startup/linux/CMakeLists.txt

```diff
@@ -27,18 +70,24 @@ function(add_startup_object name)
   )
 endfunction()
 
+check_cxx_compiler_flag("-r" LIBC_LINKER_SUPPORTS_RELOCATABLE)
+
+if(NOT LIBC_LINKER_SUPPORTS_RELOCATABLE)
+  message(STATUS "Skipping startup for target architecture ${LIBC_TARGET_ARCHITECTURE}: linker does not support -r")
```

Comment:
Do we no longer export any startup files depending on this flag?

Reply:
If `crt1.o` is not available, then `_start` will be missing. I don't think `libc-startup` will be usable then.